kolibri/content/fixtures/content_test.json

```diff
@@ -1,764 +1 @@
mat", "pk": "avi", "fields": {}}, {"model": "content.fileformat", "pk": "mp3", "fields": {}}, {"model": "content.fileformat", "pk": "mp4", "fields": {}}, {"model": "content.fileformat", "pk": "pdf", "fields": {}}, {"model": "content.fileformat", "pk": "png", "fields": {}}, {"model": "content.fileformat", "pk": "ssf", "fields": {}}, {"model": "content.formatpreset", "pk": 1, "fields": {"name": "high_res_video", "multi_language": true, "supplementary": false, "order": null, "kind": 3, "allowed_format": ["avi", "mp4"]}}, {"model": "content.formatpreset", "pk": 2, "fields": {"name": "low_res_video", "multi_language": true, "supplementary": false, "order": null, "kind": 3, "allowed_format": ["avi", "mp4"]}}, {"model": "content.formatpreset", "pk": 3, "fields": {"name": "vector_video", "multi_language": true, "supplementary": false, "order": null, "kind": 3, "allowed_format": []}}, {"model": "content.formatpreset", "pk": 4, "fields": {"name": "es_subtitle", "multi_language": false, "supplementary": true, "order": null, "kind": 3, "allowed_format": ["ssf"]}}, {"model": "content.formatpreset", "pk": 5, "fields": {"name": "thumbnail", "multi_language": false, "supplementary": true, "order": null, "kind": 3, "allowed_format": ["png"]}}, {"model": "content.language", "pk": "en", "fields": {"lang_name": "English"}}, {"model": "content.language", "pk": "es", "fields": {"lang_name": "Spanish"}}, {"model": "content.language", "pk": "zh", "fields": {"lang_name": "Chinese"}}, {"model": "content.file", "pk": 1, "fields": {"checksum": null, "available": false, "file_size": null, "content_copy": "", "contentmetadata": 2, "file_format": "mp4", "preset": 1, "lang": "en"}}, {"model": "content.file", "pk": 2, "fields": {"checksum": null, "available": false, "file_size": null, "content_copy": "", "contentmetadata": 2, "file_format": "avi", "preset": 2, "lang": "en"}}, {"model": "content.file", "pk": 3, "fields": {"checksum": null, "available": false, "file_size": null, "content_copy": "", "contentmetadata": 4, "file_format": null, "preset": 3, "lang": "es"}}, {"model": "content.file", "pk": 4, "fields": {"checksum": null, "available": false, "file_size": null, "content_copy": "", "contentmetadata": 4, "file_format": "avi", "preset": 2, "lang": null}}, {"model": "content.file", "pk": 5, "fields": {"checksum": null, "available": false, "file_size": null, "content_copy": "", "contentmetadata": 4, "file_format": "png", "preset": 5, "lang": null}}, {"model": "content.license", "pk": 1, "fields": {"license_name": "WTFPL"}}, {"model": "content.license", "pk": 2, "fields": {"license_name": "GNU"}}, {"model": "content.license", "pk": 3, "fields": {"license_name": "CC"}}, {"model": "content.license", "pk": 4, "fields": {"license_name": "MIT"}}, {"model": "content.prerequisitecontentrelationship", "pk": 1, "fields": {"contentmetadata_1": 1, "contentmetadata_2": 2}}, {"model": "content.relatedcontentrelationship", "pk": 1, "fields": {"contentmetadata_1": 2, "contentmetadata_2": 3}}]
```

Comment:
Any chance we can use different slug names than things like [`slut_1`](https://translate.google.com/#en/zh-CN/slut)?